.flake8

```diff
@@ -1,9 +1,12 @@
 [flake8]
-extend-exclude = .direnv,.venv,venv
+extend-exclude = .direnv,.github,.ipynb_checkpoints,.pytest_cache,.venv,coverage,htmlcov,venv
 extend-select = \
     W504  # match black&PEP8 putting binary operators after new lines
-ignore = \
-    E203 \ # whitespace before : (black disagrees)
-    E501 \ # line too long (black fixes long lines, except for long strings which may benefit from being long (eg URLs))
-    W503   # line break before binary operator (black disagrees)
+ignore =
+    # whitespace before : (black disagrees)
+    E203
+    # line too long (black fixes long lines, except for long strings which may benefit from being long (eg URLs))
+    E501
+    # line break before binary operator (black disagrees)
+    W503
```

Comment:
According to [the black docs](https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#configuration) we don't need to exclude this.

Reply:
The docs are worded a little unfortunately… they're saying "[W503] should not be enabled in your [flake8] configuration.", which is why we're ignoring it here.

It's enabled by default now which is a little unfortunate.

Reply:
Oh right, I need to learn to read better.  Thanks!